pytezos/operation/group.py

```diff
@@ -302,7 +303,7 @@ def inject(self, _async=True, preapply=True, check_result=True,
                     pending_opg = self.shell.mempool.pending_operations[opg_hash]
                     if not OperationResult.is_applied(pending_opg):
                         raise RpcError.from_errors(OperationResult.errors(pending_opg))
-                    print(f'Still in mempool: {opg_hash}')
+                    logger.info('Still in mempool: %s' % opg_hash)
```

Comment:
Why not f-strings?

Reply:
There are several reasons not to use f-strings in logging statements. Firstly, they will be interpolated regardless of loglevel so lots of debug statements could hit performance hard. Secondly, error tracking tools like Sentry are unable to correctly group f-strings formatted messages. There's more nuances, check [this pylint issue](https://github.com/PyCQA/pylint/issues/2395) for the details. However if readability is the main goal I could easily revert everything back.